src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/automatic-remove/page.tsx

```diff
@@ -2,153 +2,50 @@
-                  {
-                    data_broker_count: dataBrokerCount,
-                  }
-                )}
-              </li>
-              <li>
-                {l10n.getString(
-                  "fix-flow-data-broker-profiles-automatic-remove-features-remove-personal-info"
-                )}
-              </li>
-              <li>
-                {l10n.getString(
-                  "fix-flow-data-broker-profiles-automatic-remove-features-guided-experience"
-                )}
-              </li>
-              <li>
-                {l10n.getString(
-                  "fix-flow-data-broker-profiles-automatic-remove-features-continuous-monitoring"
-                )}
-              </li>
-              <li>
-                {l10n.getString(
-                  "fix-flow-data-broker-profiles-automatic-remove-features-breach-alerts"
-                )}
-              </li>
-            </ul>
-          </div>
-          {/* Plan select */}
-          <div className={styles.selectedPlan}>
-            <strong>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-headline"
-              )}
-              <small>
-                {selectedPlanIsYearly
-                  ? l10n.getString(
-                      "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-yearly-frequency"
-                    )
-                  : l10n.getString(
-                      "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-monthly-frequency"
-                    )}
-              </small>
-            </strong>
-            {/* Price */}
-            <span>
-              {selectedPlanIsYearly
-                ? l10n.getString(
-                    "fix-flow-data-broker-profiles-automatic-remove-features-price",
-                    { price: "X.XX" }
-                  )
-                : l10n.getString(
-                    "fix-flow-data-broker-profiles-automatic-remove-features-price",
-                    { price: "X.XX" }
-                  )}
-            </span>
-            <Button
-              variant="primary"
-              href={getPremiumSubscriptionUrl({
-                type: selectedPlanIsYearly ? "yearly" : "monthly",
-              })}
-              onPress={() => (window.location.href = "../../subscribed")} // TODO replace with final UI
-            >
-              {selectedPlanIsYearly
-                ? l10n.getString(
-                    "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-yearly-button"
-                  )
-                : l10n.getString(
-                    "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-monthly-button"
-                  )}
-            </Button>
-          </div>
-        </div>
-      </div>
-    </div>
+    <AutomaticRemoveView
+      data={data}
+      subscriberEmails={subscriberEmails}
+      nextStepHref={nextStep?.href ?? ""}
```

Comment:
It'll be a link that doesn't do anything, I think. I also considered making `getNextStep`'s type indicate that it would never return `null`, because it shouldn't - but figured it might be better to force calling code to handle it just in case. Though given that this is the way I'm handling it everywhere, it might be better to just have it return an "invalid" step with the `href` set to undefined?

Reply:
Went ahead and did that in 155c0010e.